enterprise/models.py

```diff
@@ -300,6 +302,9 @@ class PendingEnterpriseCustomerUser(TimeStampedModel):
     enterprise_customer = models.ForeignKey(EnterpriseCustomer, blank=False, null=False)
     user_email = models.EmailField(null=False, blank=False, unique=True)
 
+    class Meta(object):
+        app_label = 'enterprise'
```

Comment:
It does seem odd that the system is complaining about this particular set of models.  We don't seem to have to do this with other apps/packages -- I wonder if we're missing something?